src/main/java/gregtech/common/pipelike/optical/net/OpticalNetHandler.java

```diff
@@ -74,7 +74,7 @@ private void setPipesActive() {
     }
 
     private boolean isNetInvalidForTraversal() {
-        return net == null || pipe == null || pipe.isInvalid() || pipe.isFaceBlocked(facing);
+        return net == null || pipe == null || pipe.isInvalid();
```

Comment:
Why is this check being removed?

Reply:
Optical and laser pipes could have blocked connections, when it didn't make sense to. They now are not able to block connections so this check is irrelevant